modules/user_profiles/src/Commands/UserProfilesCommands.php

```diff
@@ -0,0 +1,197 @@
+     * @param   $to_user_id user id to merge to
+     * @aliases mergeUser
+     * @usage   user_profiles:mergeUser
+     */
+    public function mergeUser(string $from_user_id, string $to_user_id)
+    {
+        
+        // $flagService = \Drupal::service('flag');
+        // $this->output()->writeln(print_r($flagService->getAllFlags(), true));
+
+        $this->output()->writeln("------------- Merge user $from_user_id into $to_user_id ---------------------------------");
+
+        $user_from = User::load($from_user_id);
+        $user_to = User::load($to_user_id);
+
+        if (!$user_from) {
+            $this->output()->writeln("  *** No user found with id $from_user_id");
+            return;
+        }
+        if (!$user_to) {
+            $this->output()->writeln("  *** No user found with id $to_user_id");
+            return;
+        }
+
+        $first_name1 = $user_from->get('field_user_first_name')->getString();
+        $last_name1 = $user_from->get('field_user_last_name')->getString();
+        $first_name2 = $user_to->get('field_user_first_name')->getString();
+        $last_name2 = $user_to->get('field_user_last_name')->getString();
+
+        $this->output()->writeln("  Merging from '$first_name1 $last_name1' to '$first_name2 $last_name2'");
+
+        $this->mergeAfffinityGroups($user_from, $user_to);
+        $this->mergeFlag('interest', $user_from, $user_to);
+        $this->mergeFlag('skill', $user_from, $user_to);
+    }
+
+    private function mergeFlag($flag_name, $user_from, $user_to)
+    {
+        $this->output()->writeln("Merging " . $flag_name . "s");
+        
+        $term = \Drupal::database()->select('flagging', 'fl');
+        $term->condition('fl.uid', $user_from->id());
+        $term->condition('fl.flag_id', $flag_name);
+        $term->fields('fl', ['entity_id']);
+        $flagged_items = $term->execute()->fetchCol();
+        if ($flagged_items == NULL) {
+            $this->output()->writeln("  From-user has no reported " . $flag_name . "s");
+            return;
+        }
+        
+        foreach ($flagged_items as $flagged_item) {
+            $term = \Drupal\taxonomy\Entity\Term::load($flagged_item);
+            $title = $term->get('name')->value;
+
+            $this->output()->writeln("  from-user has $flag_name '$title'");
+
+            // Check if already flagged. If not, set the flag.
+            $flag_service = \Drupal::service('flag');
+            $flag = $flag_service->getFlagById($flag_name);
+            $flag_status = $flag_service->getFlagging($flag, $term, $user_to);
+            if (!$flag_status) {
+                $this->output()->writeln("    Add $flag_name '$title' to to-user");
+                $flag_service->flag($flag, $term, $user_to);
+            } else {
+                $this->output()->writeln("    To-user already has $flag_name '$title'");
+            }
+        }
+
+    }
+
+    private function mergeAfffinityGroups($user_from, $user_to)
```

Comment:
Affinity Group membership is done by flagging. Can we use the mergeFlag function above?